docs/pages/Research.md

```diff
@@ -58,6 +59,91 @@ Kevin Raeder, raeder@ucar.edu</td>
 </tbody>
 </table>
 
+<span id="mpasatm" class="anchor"></span> 
+-----
+
+### Model for Prediction Across Scales (MPAS)
+
+<table>
+<colgroup>
+<col style="width: 40%" />
+<col style="width: 60%" />
+</colgroup>
+<tbody>
+<tr>
+<td><img src="../images/MPAS_spaghetti_Z500mb_anal.2008090512.png" alt="" /><br />
+<span style="font-style: italic;"> Spaghetti diagram of the ensemble analysis in terms of geopotential height at 500 mb valid at 12Z 5 Dec 2008.</span>
+</td>
+<td>
+Data assimilation for MPAS is available as an ensemble Kalman filter (EnKF)
+implemented through (Data Assimilation Research Testbed)[http://dart.ucar.edu]
+
+This work is done in a collaborative effort between the 
+[Mesoscale and Microscale Meteorology (MMM)](http://www.mmm.ucar.edu)
+Division and the DART development team. DART support for both MPAS-Atmosphere 
+and MPAS-Ocean are available as part of the standard DART package.
+
+In collaboration with NOAA's <a href="http://www.esrl.noaa.gov">Earth System Research Laboratory</a>, 
+related efforts are also underway to explore ensemble data assimilation for MPAS-Atmosphere with the 
+Gridpoint Statistical Interpolation (GSI) scheme that is operational at 
+<a href="http://www.ncep.noaa.gov">the National Centers for Environmental Prediction</a>.
+</td>
+
+</tr>
+</table>
+
+<table style="text-align:" border="0" cellpadding="2" cellspacing="2">
+<colgroup>
+<col style="width: 40%" />
+<col style="width: 60%" />
+</colgroup>
+<tbody>
+<tr>
+<td>
+<img src="../images/MPAS_grid_components.png" alt="" />
+Grid structure used in the MPAS/DART interface
+</td>
+<td style="vertical-align: top">The MPAS/DART interface is
+built on MPAS's unstructured centroidal Voronoi mesh, 
+and does not regrid to latitude-longitude grids. In MPAS, the finite-volume 
+approach based on a C-grid staggering retains prognostic equations for mass 
+at the center of finite-volume cells and for the normal component of velocity
+(<i>u</i>) at the faces (or edges in 2D) of the cells. 
+The normal component of velocity (<i>u</i>) at cell edges is then used to 
+reconstruct zonal and meridional winds at cell centers 
+(<img src="../images/MPAS_V_vector.png" style="width: 20px; height: 22px;" alt="" />)
```

Comment:
For consistency, should we put (<i>v</i>) in this line instead of trying to link to the vector "v" png?

Reply:
Guess what? Because its in an html table, none of the markdown syntax for highlighting/bolding works - I guess this is not too surprising.  So - I just reverted to html ...
<strong><i>V</i></strong>
There is no overbar to denote a vector, but it should get the point across. If this is acceptable, let me know and I'll remove the images/MPAS_V_vector.png file from the repo.